python/feathub/processors/flink/table_builder/aggregation_utils.py

```diff
@@ -41,12 +41,16 @@ def __init__(
         expr: str,
         agg_func: AggFunc,
         window_size: timedelta,
+        limit: int,
+        filter_expr: str,
```

Comment:
Given that filter_expr can be none, would it be more readable to make it `Optional[str]`?